### ISSUE TITLE
Want to add practice resources under web development section

### ISSUE THREAD

**Issue by pooja-sanap-53** [DSC-JSS-NOIDA/QuickLearn#286]
I want to add all the websites where we can practice front end development skills, and also take and complete challenges on the same under the web development section. 
Example : Websites like Codepen, Frontendmentor. 

Note: I want to contribute as a GSSOC 22 contributor.

**Comment by the-macson** [comment]
@rucha0702 Please review this issue
I think it's good idea

**Comment by rucha0702** [comment]
Can add

**Comment by the-macson** [comment]
ok @rucha0702 
I am assigning this issue to @pooja-sanap-53

**Comment by pooja-sanap-53** [comment]
Thank you for assigning this issue to me. I have started working on this .

**Comment by rucha0702** [comment]
Is this done?

**Comment by pooja-sanap-53** [comment]
In progress, will finish asap.

**Comment by vedsom** [comment]
@rucha0702, @the-macson  is this issue completed ?? if not please assign me i will do it today itself.

**Comment by pooja-sanap-53** [comment]
Yes, it is done. I have made a PR regarding the same. PR. Number -#386.
